programs/ziti-edge-tunnel/CMakeLists.txt

```diff
@@ -33,6 +33,12 @@ endif ()
 add_executable(ziti-edge-tunnel ziti-edge-tunnel.c ${NETIF_DRIVER_SOURCE} dnsmasq_manager.c ${ZITI_INSTANCE_COMMON} ${ZITI_INSTANCE_OS})
 set_property(TARGET ziti-edge-tunnel PROPERTY C_STANDARD 11)
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    find_package(PkgConfig REQUIRED)
```

Comment:
we cannot require systemd

Reply:
Moved to conditional linking. Added a feature toggle to disable feature when desired.